src/main/java/com/epam/edumanagementsystem/admin/model/entity/SuperAdmin.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.edumanagementsystem.admin.model.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table
+public class SuperAdmin {
+    @Id
```

Comment:
Add @GeneratedValue annotation besides of @Id

Reply:
ok